osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/RtuDeviceService.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.domain.da.application.services;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import org.opensmartgridplatform.domain.core.entities.DeviceAuthorization;
+import org.opensmartgridplatform.domain.core.entities.DeviceModel;
+import org.opensmartgridplatform.domain.core.entities.Manufacturer;
+import org.opensmartgridplatform.domain.core.entities.Organisation;
+import org.opensmartgridplatform.domain.core.entities.ProtocolInfo;
+import org.opensmartgridplatform.domain.core.entities.Ssld;
+import org.opensmartgridplatform.domain.core.repositories.DeviceAuthorizationRepository;
+import org.opensmartgridplatform.domain.core.repositories.DeviceModelRepository;
+import org.opensmartgridplatform.domain.core.repositories.ManufacturerRepository;
+import org.opensmartgridplatform.domain.core.repositories.OrganisationRepository;
+import org.opensmartgridplatform.domain.core.repositories.ProtocolInfoRepository;
+import org.opensmartgridplatform.domain.core.repositories.RtuDeviceRepository;
+import org.opensmartgridplatform.domain.core.valueobjects.AddRtuDeviceRequest;
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceFunctionGroup;
+import org.opensmartgridplatform.domain.core.valueobjects.RtuDevice;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalExceptionType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service(value = "domainDistributionAutomationRtuDeviceService")
+@Transactional(value = "transactionManager")
+public class RtuDeviceService {
+
+    private static final String LOCAL_HOST = "127.0.0.1";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1313)

Reply:
No.

Reply:
Hah, SonarQube is always complaining about this. Can be suppressed using `@SuppressWarnings("squid:S1313")`